contracts/VoucherHub.sol

```diff
@@ -253,6 +253,37 @@ contract VoucherHub is
         emit VoucherRefunded(msg.sender, amount);
     }
 
+    /**
+     * Drain funds from voucher if it is expired. Transfer all RLC balance
+     * on PoCo from voucher to voucherHub and burn all credits.
+     * @param voucher address of the expired voucher to drain
+     */
+    function drainVoucher(address voucher) external onlyRole(VOUCHER_MANAGER_ROLE) {
+        VoucherHubStorage storage $ = _getVoucherHubStorage();
+        require($._isVoucher[voucher], "VoucherHub: unknown voucher");
+        uint256 amount = balanceOf(voucher);
+        Voucher(voucher).drain(amount);
+        _burn(voucher, amount);
+        emit VoucherDrained(voucher, amount);
+    }
+
+    /**
+     * Withdraw full balance of this contract from PoCo and send it to the
+     * specified address.
+     * @param receiver address that will receive withdrawn funds
+     */
+    function withdraw(address receiver) external onlyRole(ASSET_ELIGIBILITY_MANAGER_ROLE) {
```

Comment:
Maybe assert receiver it not empty to avoid human error?
I wonder if we should not set an amount to withdraw, ex. the hub has 5K RLC, we now want to set it back to 1K RLC